### DISCUSSION TITLE
[Feature]: Partner Sharing aka Sharing whole library with other user

### DISCUSSION THREAD

**Discussion by anpavlov** [immich-app/immich#1668] [Feature Request]
### Feature detail

It would be great to see a possibility to share whole library with another user, so that they could view my all photos without editing (uploading into my library).

### Platform

Server

**Comment by axgd-code** [comment]
Is it related to these issues ?
- https://github.com/immich-app/immich/issues/628
- https://github.com/immich-app/immich/issues/265

**Comment by anpavlov** [comment]
I don't think so. My suggestion is about sharing within single immich instance. My main reason is ability to share library with my wife, so that she can view my photo library anytime, while i can view her library.

**Comment by sbkg0002** [comment]
> I don't think so. My suggestion is about sharing within single immich instance. My main reason is ability to share library with my wife, so that she can view my photo library anytime, while i can view her library.

For me this is a feature to be able to replace Gphotos.

**Comment by chasewinnings391** [comment]
I'm currently sharing my immich account with my partner as a workaound. Main downside of this is that when I delete a photo which was taken by my partner, it will automatically be re-uploaded again from her phone.

**Comment by Numeri** [comment]
What is the current status of this feature? The original issue was marked done, but the button for sharing with a partner is still greyed-out for me.

**Reply by bo0tzz** [reply]
The issue is closed because all feature requests were converted to discussions, it hasn't been implemented yet.

**Comment by ja2ui0** [comment]
Glomming onto this discussion, I went to "share with partner" in the android app and I thought I was doing something wrong that it was greyed out. My wife and I are both all-in on Immich now and being able to share entire libraries with each other and collaborate on albums is a crucial feature for us, very much looking forward to it!

It may or may not be a different feature request (if so, LMK and I'll open it), but should not be limited to a single partner. Ideally there would be "family sharing" where perhaps entire libraries could be shared with an arbitrary number of people on a RO / RW basis.

If there's a middle ground, I would hope that partner sharing would not be limited to two individuals at least.

**Comment by alextran1502** [comment]
Implemented in #2388
